include/sys/trace_acl.h

```diff
@@ -92,7 +92,7 @@ DECLARE_EVENT_CLASS(zfs_ace_class,
 	    __entry->z_blksz		= zn->z_blksz;
 	    __entry->z_seq		= zn->z_seq;
 	    __entry->z_mapcnt		= zn->z_mapcnt;
-	    __entry->z_gen		= zn->z_gen;
+	    __entry->z_gen		= zn->z_inode.i_generation;
```

Comment:
Since these values have now been unified `z_gen` should be dropped entirely from the trace file.  We're already logging i_generation.